### ISSUE TITLE
AttributeError: 'DockerManager' object has no attribute 'client'

### ISSUE THREAD

**Issue by qwebot** [ansible/ansible-modules-core#4557]
##### ISSUE TYPE

- Bug Report
##### COMPONENT NAME

docker module
##### ANSIBLE VERSION

ansible 2.1.1.0
##### DOCKER VERSION

1.11.2
##### DOCKER-PY VERSION

docker-py version: 1.5.0
##### STEPS TO REPRODUCE

```
  docker:
    name: "name"
    image: "docker/php/project:version"
    detach: false
    state: absent
    pull: always
    registry: "{{ docker_registry.url }}"
    username: "{{ docker_registry.username }}"
    password: "{{ docker_registry.password }}"
    env_file: ./env_files/file.env
```

##### ACTUAL RESULTS

```
FAILED! => {"changed": false, "failed": true, "module_stderr": "", "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_zFiTa0/ansible_module_docker.py\", line 1975, in <module>\r\n    main()\r\n  File \"/tmp/ansible_zFiTa0/ansible_module_docker.py\", line 1912, in main\r\n    manager = DockerManager(module)\r\n  File \"/tmp/ansible_zFiTa0/ansible_module_docker.py\", line 749, in __init__\r\n    self.environment = self.get_environment(env, env_file)\r\n  File \"/tmp/ansible_zFiTa0/ansible_module_docker.py\", line 895, in get_environment\r\n    self.ensure_capability('env_file')\r\n  File \"/tmp/ansible_zFiTa0/ansible_module_docker.py\", line 870, in ensure_capability\r\n    self._check_capabilities()\r\n  File \"/tmp/ansible_zFiTa0/ansible_module_docker.py\", line 853, in _check_capabilities\r\n    api_version = self.client.version()['ApiVersion']\r\nAttributeError: 'DockerManager' object has no attribute 'client'\r\n", "msg": "MODULE FAILURE", "parsed": false}
```

Problem with parameter env_file. If we set key => value in env parameter it's all ok

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by zfil** [comment]
wontfix
this module is deprecated
@cove, @softzilla, @zfil, @ThomasSteinbach guys, in general I suggest we stop docker modules needing docker-py to rely only on native docker component (like compose)
